client/src/app/site/pages/meetings/services/media-manage.service.ts

```diff
@@ -68,8 +72,12 @@ export class MediaManageService {
     }
 
     public getLogoUrl(place: LogoPlace): string | null {
-        // Note: we are not fetching the mediafile view model at any place.
-        const mediafileId = this.activeMeetingService.meeting?.logo_id(place);
+        // Note: we are not fetching the mediafile view model at any place except when filtering for the defaults.
+        const mediafileId =
+            (this.activeMeetingService.meetingId
+                ? this.activeMeetingService.meeting?.logo_id(place)
+                : this.getGeneralMediafileIdByToken(`orga`, place)) ??
+            this.getGeneralMediafileIdByToken(`global`, place);
```

Comment:
Take orga out of this calculation (it can't be filled).
Also remove the replacement with global default if there is no id, instead put separate check for global default into `else if ` block  if `mediafileId` is empty